lms/djangoapps/courseware/features/lti.py

```diff
@@ -129,7 +132,7 @@ def incorrect_lti_is_rendered(_step):
 
 @step('the course has correct LTI credentials with registered (.*)$')
 def set_correct_lti_passport(_step, user='Instructor'):
-    coursenum = 'test_course'
+    coursenum = TEST_COURSE_NAME
```

Comment:
I don't think it should stop this PR from merging as it was this way before, but thought I'd point out that this variable name seems confusing.